docs/performance/Basics.md

```diff
@@ -87,15 +87,15 @@ As the drop in price is accounted for by the dividend represented by `ExpectedIn
 
 ## Partial Withdrawals
 
-Another scenario that could cause this is that of a partial withdrawal.
+Another scenario that could cause this fluctuation in performance is that of a partial withdrawal. Ignoring ExpectedIncome it would look like the following:
 
-|Day|StartValue|EndValue|Cash flow|Performance Gross|
-|--------|--------|--------|--------|--------|
-|Day n|£0|£100|-|-|
-|Day n+1|£100|£100|-|((100 + 0 - 0) / 100) - 1 = 0 or 0%|
-|Day n+2|£100|£90|-|((90 - 0) / 100) - 1 = -0.1 or -10%|
```

Comment:
Why has this negative sign been removed?

Reply:
Copy & Paste error; fixed now